recipe/meta.yaml

```diff
@@ -11,48 +11,46 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: True  # [not linux]
+  number: 1
+  skip: True  # [win]
 
 requirements:
   build:
-    - {{ compiler('cxx') }}   # [linux] required for C++ headers (when using clang)
-    - {{ compiler('c') }}     # [linux] (when using gcc, disable it when switching to clang)
+    # In OSX, we use clang++ from clangdev as a compiler
+    - {{ compiler('cxx') }}   # [not osx]
+    - {{ compiler('c') }}     # [not osx]
+    - boost-cpp 1.68.*        # to avoid cmake picking up boost-cpp 1.70+
     - cmake
     - make
-    - maven
-    - openjdk 8.*
-    - wget
-    - clangdev      # [linux] llvm 8 is ok when using gcc as compiler
-    - clangdev 7.*  # [osx] using clang 8 as compiler, many tests fail 
+    #- openjdk 8.*
+    #- llvmdev                 # [linux]
+    #- clangdev                # [linux]
+    #- wget                    # [linux]
     - go
-    - bisonpp
+    - maven
   host:
-    - snappy
-    - arrow-cpp >=0.12.1
-    - blosc
     - boost-cpp 1.68.*
+    - llvmdev
+    - clangdev
     - double-conversion
-    - libgdal 2.*   # with libgdal-3 one ImportTest fails
-    - flex
+    - snappy
     - gflags
     - glog
     - libarchive
     - libkml
     - libpng
-    - llvmdev       # [linux]
-    - clangdev      # [linux], required here for clang/llvm 8 support
-    - llvmdev 7.*   # [osx], see above
-    - clangdev 7.*  # [osx], see above
-    - openssl
+    - blosc
+    - libgdal 2.*   # with libgdal-3 one ImportTest fails
+    - arrow-cpp >=0.12.1
     - thrift-cpp >=0.11.0
     - ncurses
-    # omnscidb server calls clang++ in runtime for UDF support
-    - {{ compiler('cxx') }}   # [linux] required for C++ headers (when using clang)
```

Comment:
`fatal error: 'cassert' file not found`

not sure if ir could be failing because `compiler(cxx)` was removed.

Reply:
Fixed.